### DISCUSSION TITLE
How to hide Footer Actions in resource pages (Create, Edit, View)

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9116] [Help] [answered]
<img width="1192" alt="Screenshot 2023-10-15 at 17 53 55" src="https://github.com/filamentphp/filament/assets/121255432/f3235c00-5254-409a-ae0a-f497bc153489">

**Comment by leandrocfe** [comment, marked as answer]
You can add this method on the CreatePage, Edit and View

```php
protected function getFormActions(): array
{
    return [];
}
```

**Reply by devhoussam1998** [reply]
thank you @leandrocfe
